airbyte-integrations/connectors/source-okta/source_okta/components.py

```diff
@@ -52,6 +55,7 @@ def get_refresh_request_body(self) -> Mapping[str, Any]:
         }
 
     def refresh_access_token(self) -> Tuple[str, int]:
+        print(self.token_refresh_endpoint)
```

Comment:
If you do have logging here, you probably don't want a rogue print statement. Use logging all the way?